config.go

```diff
@@ -30,11 +30,24 @@ type WAFConfig interface {
 	// WithContentInjection enables content injection.
 	WithContentInjection() WAFConfig
 
-	// WithRequestBodyAccess configures access to the request body.
-	WithRequestBodyAccess(config RequestBodyConfig) WAFConfig
+	// WithRequestBodyAccess enables access to the request body.
+	WithRequestBodyAccess() WAFConfig
 
-	// WithResponseBodyAccess configures access to the response body.
-	WithResponseBodyAccess(config ResponseBodyConfig) WAFConfig
+	// WithRequestBodyLimit sets the maximum number of bytes that can be read from the request body. Bytes beyond that set
+	// in WithInMemoryLimit will be buffered to disk.
+	WithRequestBodyLimit(limit int) WAFConfig
```

Comment:
- do we need to use `int`? why not using uint instead? Is there a reason to use negative sizes?
- maybe rename the methods to `WithRequestBodyBytesLimit` instead of just "Limit"?

Reply:
we should always avoid numeric conversion. For all ints that are no bigger than INT64 we should always stick to int

Reply:
I understand that we want to avoid numeric conversion. But is there a rule on not using a type designed for positives? Also, int64 is not int. Do we want to throw away the type safety of a value that cannot be negative because it is "easier" to use?